app/models/searchworks/holdings.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Searchworks
+  ###
+  #  ReqestedHoldings winnows down the entire
```

Comment:
copypasta? this is no longer called `RequestedHoldings`

Reply:
Fixed.